library/src/main/java/com/novoda/downloadmanager/CompletedDownloadBatchesExtractor.java

```diff
@@ -8,7 +8,7 @@
 import java.util.List;
 import java.util.Set;
 
-class MigrationExtractor {
+public class CompletedDownloadBatchesExtractor {
```

Comment:
As this is now the responsibility of the client application this class will be moved to the `demo` in the next PR.